src/lib/server/alerts.ts

```
@@ -0,0 +1,78 @@
+	try {
+		await fetch(url, {
+			method: 'POST',
+			headers: { 'Content-Type': 'application/json' },
+			body: JSON.stringify({ chat_id: cfg.chatId, text }),
+			signal: AbortSignal.timeout(3000)
+		});
+	} catch (err) {
+		// Logged by caller (rpcMetrics.reportChainExhausted) so the alert-delivery-failure
+		// shows up next to the chain-exhausted log line. Rethrow so caller can record it.
+		throw err;
+	}
```

ESLint

[error] Unnecessary try/catch wrapper.

(no-useless-catch)